docs/scaling/multi-node-setup.rst

```diff
@@ -4,162 +4,237 @@
-
-+-----------------+-----------+---------------------------------------+
-| CrateDB Version | Reference | Configuration Example                 |
-+=================+===========+=======================================+
-| <=4.x           | `latest`_ | .. code-block:: yaml                  |
-|                 |           |                                       |
-|                 |           |     discovery.seed_hosts:             |
-|                 |           |       - node1.example.com:4300        |
-|                 |           |       - node2.example.com:4300        |
-|                 |           |       - 10.0.1.102:4300               |
-|                 |           |       - 10.0.1.103:4300               |
-+-----------------+-----------+---------------------------------------+
-| <=3.x           | `3.3`_    | .. code-block:: yaml                  |
-|                 |           |                                       |
-|                 |           |     discovery.zen.ping.unicast.hosts: |
-|                 |           |       - node1.example.com:4300        |
-|                 |           |       - node2.example.com:4300        |
-|                 |           |       - 10.0.1.102:4300               |
-|                 |           |       - 10.0.1.103:4300               |
-+-----------------+-----------+---------------------------------------+
 
-.. NOTE::
+.. _manual-bootstrapping:
 
-   When adding new nodes to the cluster, you do not need to update the list of
-   hosts for existing/running nodes. The cluster will find and add new
-   nodes when they ping existing ones.
+Manual bootstrapping
+--------------------
 
-Providing a list of expected hosts is just one node discovery mechanism.
+To run a CrateDB cluster across multiple hosts, you must manually configure the
+bootstrapping process by telling nodes to:
 
-CrateDB also supports node discovery via DNS as well as discovery via API for
-clusters running on *Amazon Web Services* (AWS) or Microsoft Azure. See `the
-documentation <https://crate.io/docs/reference/configuration.html#discovery>`_
-for more information.
+  a. :ref:`Discover other nodes <discovery>`, and
+  b. :ref:`Elect a master node <master-node-election>`
 
-.. _master-node-election:
+You must also configure the :ref:`metadata gateway <metadata-gateway>` (as with
+auto-bootstrapping).
 
-Master node election
-====================
 
-In a CrateDB cluster, the master node is responsible for making changes to the
-global cluster state. The master node is elected as part of the `master node
-election`_.
+.. _discovery:
 
-CrateDB requires a *quorum* of nodes in the cluster before a master can be
-elected. This insures that multiple masters are not elected in the event of a
-network partition (which would lead to a `split-brain`_ scenario).
+Discovery
+~~~~~~~~~
 
-.. _master-node-election-4x:
+With CrateDB 4.x and above, you can configure a list of master-eligible nodes
```

Comment:
The nodes here are not necessarily the `master-eligible` nodes, but this can include any kind of nodes.